### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by power-pole03**
Changed a background and put a logo video

### CODE REVIEW

File: src/Home.jsx
Comment (ryanprice) [inline src/Home.jsx:1]: Can you use the .webm version of the file I provided ?

**Review by ryanprice — CHANGES_REQUESTED** [review]

File: src/Home.jsx
Comment (ryanprice) [reply src/Home.jsx:1]: to replace the .mp4 file

Comment (power-pole03) [reply src/Home.jsx:1]: Changed the mp4 to webm

**Review by ryanprice — APPROVED** [review]
great

**Pull request merged**